packages/suite/src/views/settings/SettingsCoins/SettingsCoins.tsx

```diff
@@ -19,8 +20,11 @@ const StyledSettingsLayout = styled(SettingsLayout)`
 `;
 
 export const SettingsCoins = () => {
+    const dispatch = useDispatch();
```

Comment:
nit: there's [an article](https://www.notion.so/satoshilabs/Components-9159b639d21b43ef8b66be150854b6b6?pvs=4#308aa7db519e4f4eaf70312f60ce030a) in the style guide regarding component structure. It's not obligatory to follow but I believe using a somewhat similar pattern leads to a massive increase in readability across the codebase! Having global and local state at the very top IMO makes the most sense since the components are functions of state and props.